app/views/admin/users/_form.html.erb

```diff
@@ -19,11 +19,25 @@
     </div>
     <div class="col-md-6">
       <h3>Neighbourhoods</h3>
-      <p>Grants access to every partner and calendar in these neighbourhoods. Use with care!</p>
-      <%= f.association :neighbourhoods,
-                        label: false,
-                        collection: options_for_neighbourhoods,
-                        input_html: {class: 'form-control select2' } %>
+      <p>You have <%= pluralize @user.neighbourhoods.count, 'neighbourhood' %>.</p>
```

Comment:
this doesn't really make sense because this is the form to edit the user, not the form to edit yourself